bundle/config/paths/paths.go

```diff
@@ -6,14 +6,18 @@ import (
 )
 
 type Paths struct {
-	// ConfigFilePath holds the path to the configuration file that
-	// described the resource that this type is embedded in.
-	ConfigFilePath string `json:"-" bundle:"readonly"`
+	// Absolute path on the local file system to the configuration file that holds
+	// the definition of this resource.
+	LocalConfigFilePath string `json:"-" bundle:"readonly"`
+
+	// Relative path from the bundle root to the configuration file that holds
+	// the definition of this resource.
+	ConfigFilePath string `json:"config_file_path,omitempty" bundle:"readonly"`
```

Comment:
To make the refactoring safer and smaller you could call it `ConfigFilePathRelative`, then you don't need to do all the rename and also make sure that nowhere else in code we accidentally misuse `ConfigFilePath ` where we want to use `LocalConfigFilePath `

@pietern wdyt?

Reply:
Ack, I am fine with either. This refactor was done because `LocalConfigFilePath` was not serialized and I would prefer to have the shorter name in the json serialization (ie `config_file_path`).

Good point about the safety. It's safe(ish) because of the IDE but git merge can mess this up.

Lets keep the `LocalConfigFilePath` and name the other field `ConfigPath` to avoid inadvertantly breaking code.

Reply:
Done. Reverted the rename. The new field is called `RelativePath`.